Add metrics-server docs (#2049)
kubeedge/kubeedge: docs/setup/keadm.md

6. Restart all the cloudcore and edgecore.

### Support Metrics-server in Cloud
1. The realization of this function point reuses cloudstream and edgestream modules. So you also need to perform all steps of *Enable `kubectl logs` Feature*.

> 2. Since the kubelet ports of edge nodes and cloud nodes are not the same, the current release version of metrics-server does not support automatic port identification, so you need to manually compile the image yourself.

Reviewer: > the current release version of metrics-server does not support automatic port identification

Better to point out exact version numbers of metrics-server that support and doesn't support

Author: Updated. Now the metrics-server is 0.3.7. This feature is estimated to be launched in 0.4.0. Some features of 0.4.0 are still in the early stages of development.

```bash
git clone https://github.com/kubernetes-sigs/metrics-server.git
cd metrics-server
make container
docker images
docker tag 2d9eb6e0d887 metrics-server-kubeedge:latest
```

3. Apply the deployment yaml. For specific deployment documents, you can refer to https://github.com/kubernetes-sigs/metrics-server/tree/master/manifests.

**IMPORTANT NOTE:**
1. Metrics-server needs to use hostnetwork network mode.

2. Use the image compiled by yourself and set imagePullPolicy to IfNotPresent.

> 3. Turn on the feature of --kubelet-use-node-status-port

Reviewer 2: ```suggestion
3. Enable the feature of --kubelet-use-node-status-port for Metrics-server
```
It shoulod be for Metrics-server here I think :)
